airbyte-cdk/bulk/toolkits/load-object-storage/src/test/kotlin/io/airbyte/cdk/load/file/object_storage/ObjectStoragePathFactoryTest.kt

```diff
@@ -196,7 +195,8 @@ class ObjectStoragePathFactoryTest {
             val prefixOnly = "prefix/$namespace/$name/"
             Assertions.assertEquals(
                 prefixOnly,
-                pathFactory.getLongestStreamConstantPrefix(stream1, false),
+                pathFactory
+                    .getFinalDirectory(stream1, substituteStreamAndNamespaceOnly = true)
```

Comment:
Nit: Test name is `testLongestConstantPrefix` yet method under test is `getFinalDirectory`

Reply:
fixed